ql/src/test/results/clientpositive/llap/compute_kll_sketch.q.out

```diff
@@ -7,22 +7,25 @@ POSTHOOK: type: QUERY
 POSTHOOK: Input: default@alltypesorc
 #### A masked pattern was here ####
 
-### KLL sketch summary:
-   K                    : 200
-   min K                : 200
-   M                    : 8
-   N                    : 9173
-   Epsilon              : 1.329%
-   Epsison PMF          : 1.652%
-   Empty                : false
-   Estimation Mode      : true
-   Levels               : 6
-   Sorted               : false
-   Buffer Capacity Items: 547
-   Retained Items       : 535
-   Storage Bytes        : 2192
-   Min Value            : -64.0
-   Max Value            : 62.0
+### KllKllFloatsSketch Summary:
```

Comment:
The name seems redundant, I guess `KllFloatSketch Summary` is better if we need to change the existing name (even if I don't see much interest, what type of KLL sketch we use it's an internal detail I'd say).

Reply:
I think this needs to change related codes in apache/datasketches. :)